crates/runtime/src/tools/builtin/web_search/mod.rs

```diff
@@ -101,13 +102,15 @@ impl SearchEngine {
     }
 }
 
+#[derive(Debug, Clone, Serialize, Deserialize, JsonSchema)]
+#[serde(untagged)]
+pub enum WebSearchParamsType {
```

Comment:
```suggestion
pub enum WebSearchParams {
```
This should become `WebSearchParams` otherwise we will have a level of nesting in the tool interface.

i.e.
```shell
curl -XPOST http://localhost:8090/v1/tools/websearch -d '{
  "params": {
    "query": "my query"
  }
}'
```